### DISCUSSION TITLE
New context menu entry Apps and Artifacts

### DISCUSSION THREAD

**Discussion by hosy** [deszip/BitBot#31] [Feature requests]
It would be great to have a context menu entry which copies the generated binary link (or install page link) into the clipboard.

**Comment by deszip** [comment]
Hi, thanks, will think about it, if Bitrise has this link in the build info - should be easy

**Comment by deszip** [comment]
Hi @hosy in v1.1.0 I've added an item to build menu that navigates you to the artefacts page. Unfortunately bitrise API doesn't provide a direct binary link.
